src/test/java/teammates/test/cases/util/RecaptchaVerifierTest.java

```diff
@@ -0,0 +1,89 @@
+package teammates.test.cases.util;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpResponseException;
+import org.testng.annotations.Test;
+
+import teammates.common.util.RecaptchaVerifier;
+import teammates.test.cases.BaseTestCase;
+
+/**
+ * SUT: {@link RecaptchaVerifier}.
+ * @see <a href="https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha-what-should-i-do">Automated testing with reCAPTCHA v2</a>
+ */
+public class RecaptchaVerifierTest extends BaseTestCase {
+
+    @Test
+    public void testIsVerificationSuccessful() {
+        ______TS("null or empty captcha response");
+        assertFalse(new RecaptchaVerifier().isVerificationSuccessful(null));
+        assertFalse(new RecaptchaVerifier().isVerificationSuccessful(""));
+
+        ______TS("Any other key"); // success when using the official secret test key
+        assertTrue(new RecaptchaVerifier().isVerificationSuccessful("testResponseKey"));
+        assertTrue(new RecaptchaVerifier().isVerificationSuccessful(".#?"));
```

Comment:
let's not make the connection with the google server. If google server goes down, we will have trouble on our side as well. That's the main point of writing stub for this test. What about we just put the fake successful responses in the stub as well?

Reply:
fixed